src/selectors/internal.js

```diff
@@ -1,3 +1,25 @@
+import { get as lodashGet } from 'lodash';
+import { fromJS, Iterable } from 'immutable';
+
+export const get = (object, path, defaultValue) => {
```

Comment:
this is used as an interop so we can call selectors on objects and immutables

Reply:
Could you name it something more specific than `get` so we can avoid confusion between using it and using the lodash version?